source/adapters/level_zero/device.cpp

```diff
@@ -485,6 +485,8 @@ ur_result_t urDeviceGetInfo(
   case UR_DEVICE_INFO_BUILT_IN_KERNELS:
     // TODO: To find out correct value
     return ReturnValue("");
+  case UR_DEVICE_INFO_LOW_POWER_EVENTS_EXP:
+    return ReturnValue(UR_DEVICE_INFO_LOW_POWER_EVENTS_EXP);
```

Comment:
the return type for `UR_DEVICE_INFO_LOW_POWER_EVENTS_EXP` is a `ur_bool_t`, but it looks like we're just returning the enum back when being queried.

Reply:
when this flag is set, we return true now.